query-builder/src/test/java/com/datastax/oss/driver/api/querybuilder/update/UpdateIdempotenceTest.java

```diff
@@ -148,4 +148,11 @@ public void should_not_be_idempotent_if_concatenating_to_collection() {
         .hasCql("UPDATE foo SET l=[1,2,3]+l WHERE k=?")
         .isNotIdempotent();
   }
+
+  @Test
+  public void should_be_idempotent_if_relation_does_not_have_right_operand() {
+    assertThat(update("foo").setColumn("col1", literal(42)).whereColumn("col2").isNotNull())
```

Comment:
could you tell me what is a `rightOperand` in this test case?

Reply:
There isn't. `IS NOT NULL` could be seen as a function of arity 1: `boolean isNotNull(x)` and thus it takes only one operand (the "left" operand). Other operators, such as `+`, are functions of arity 2: `int plus(x, y)`, which is why they have 2 operands, left (x) and right (y).